### PR TITLE
feat(request): support proxy option

### PR DESCRIPTION
**Pull request by zewa666**
this allows to pass in a proxy to be used via options. It essentially mimics the behavior of passing in the username/accesskey either via ENVs or options. Moreover this used to be available in v1 and now with the PR shouldn't be a breaking change any longer.

### CODE REVIEW

**Comment by zewa666** [comment]
proxy param to cli is done, can you clarify what you meant with the example for ENV vars? The `usage` section of the README is already mentioning that you'd need to set it up.

File: src/constants.js [hunk 1]
```diff
@@ -98,4 +98,9 @@ export const CLI_PARAMS = [{
     name: 'headless',
     default: DEFAULT_OPTIONS.headless,
     description: 'if set to true you are accessing the headless Sauce instances (this discards the `region` option)'
+}, {
+    alias: 'p',
+    name: 'proxy',
+    default: undefined,
```
Comment (christian-bromann) [inline src/constants.js]:
```suggestion
```

**Comment by zewa666** [comment]
I wasn't sure whether its a good practice to leave the default with undefined there, for better understanding. But certainly doesn't harm to remove it

**Comment by christian-bromann** [comment]
Released as `v2.3.0`

**Pull request merged**